app/src/main/java/com/trainstation/frettirtest2/MainActivity.java

```diff
@@ -14,23 +14,22 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    ViewPager pager;
-    ViewPageAdapter adapter;
+    private ViewPager pager;
+    private ViewPageAdapter adapter;
     private DrawerLayout mDrawerLayout;
     private ListView mDrawerList;
     private String[] mPlanetTitles;
 
-    NewsService newsService;
+    private NewsService newsService;
 
     public Observable<NewsResponse> getNewsResponse()
     {
         RestAdapter restAdapter = new RestAdapter.Builder().setEndpoint("https://itunes.apple.com").build();
 
         newsService = restAdapter.create(NewsService.class);
-        return newsService.getNewsData();
+        return newsService.getNewsData("beyonce", "musicVideo");
```

Comment:
look here: https://github.com/helgio/frettaApp/pull/2/files#diff-eb8dcd859b45dc80d12742a54b30d34aL13 to see what is happening